examples/04_fuzzy_joining.py

```diff
@@ -253,8 +248,8 @@
     y="Happiness score",
     lowess=True,
 )
-fig.set_ylabel("Happiness index")
```

Comment:
thanks for the fix! the figure title changes don't seem related to filtering warnings, right?
I think the old title might easier to understand for some readers, because "effect" could be understood to imply that life expectancy has a causal effect on happiness